lib/api/secure_store.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:convert';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:authenticator/model/account.dart';
+
+class SecureStore {
+  final FlutterSecureStorage storage;
+
+  SecureStore(this.storage);
```

Comment:
Yes, `storage` is generally preferred to `this.storage` when referring to fields within a class. However, the `this` keyword is required in the context of the constructor to [initialize the field](https://dart.dev/guides/language/language-tour#constructors). The other way to do it would require a constructor body, but I prefer this way because it is more concise. Which way should we use going forward?

Reply:
I'm tempted to say that it would be better to declare the constructors explicitly, but this is where my inexperience with dart comes into play. So we can leave it as is and merge.